client/src/components/footer/footer.js

```diff
@@ -135,7 +135,7 @@ class Footer extends Component {
         let display = getGameState(state) === "GAME_IDLE" || body !== 2;
         let hideOnLanding = lab === 0; // for buttons that should not be displayed on the landing page
         return (
-            <div>
+            <div className="changeFooter">
                 <div className="container" style={{display: display ? "block" : "none"}}>
                     <button
                         className="btn btn-second btn-xl text-uppercase js-scroll-trigger back "
```

Comment:
I think you may need to pull from master to get the new buttons with shorter hyphens